localization/spanish/dummy

```diff
@@ -0,0 +1 @@
+
```

Comment:
I think a dummy fine is not needed here, as you have other files in the folder.

Reply:
deleted